docusaurus/platform_versioned_docs/version-1.6/understanding-airbyte/airbyte-protocol-docker.md

```diff
@@ -51,3 +51,29 @@ The `write` command will consume `AirbyteMessage`s from STDIN.
 - Connectors receive arguments on the command line via JSON files. `e.g. --catalog catalog.json`
 - They read `AirbyteMessage`s from STDIN. The destination `write` action is the only command that consumes `AirbyteMessage`s.
 - They emit `AirbyteMessage`s on STDOUT.
+
+## Additional Docker Image Requirements
+
+### Environment variable: `AIRBYTE_ENTRYPOINT` 
+
+The Docker image must contain an environment variable called `AIRBYTE_ENTRYPOINT`. This must be the same as the `ENTRYPOINT` of the image.
+
+## Non-Root User: `airbyte`
+
+The Docker image should run under a user named `airbyte`.
+
+## Specified `/airbyte` directory
+
+The Docker image must have a directory called `/airbyte`, which the user `airbyte` owns and can write to.
+
+This is the directory to which temporary files will be mounted, including the `config.json` and `catalog.json` files.
+
+## Only write file artifacts to directories permitted by the base image
+
+The connector code must only write only to directories permitted within the connector's base image.
```

Comment:
File movement connectors definitely read/write to the staging path we pass in.

Reply:
It would be great to document the specific directory if we can. While, IIRC, the party is sent via env var, we would want to document it so the base image definition gives write access.